### DISCUSSION TITLE
I have a message "Your platform does not support this feature yet"

### DISCUSSION THREAD

**Discussion by 93041025** [ModelSurge/sd-webui-comfyui#87] [Q&A] [answered]
![스크린샷 2023-07-27 155413](https://github.com/ModelSurge/sd-webui-comfyui/assets/130146627/99099ffa-bd03-41e0-a834-1e9cb62673de)

I followed the instructions and successfully installed the extension and Comfy. However, after restarting WebUI, I encountered an error message that said, 'Your platform does not support this feature yet.' This prevented me from using Comfy with WebUI.  Comfy is still functioning properly when I connect it with the web browser. My version of WebUI is 1.5.0.

**Comment by PladsElsker** [comment]
Yes, this message only appears for WSL users. If you are not using WSL, this is not supposed to happen. 

We put this message in place because somehow, the application deadlocks on WSL. We are investigating how to mitigate this at the moment. 

You should still be able to use ComfyUI in the "ComfyUI" tab like before. Let us know if that's not the case. 

See #76

**Comment by PladsElsker** [comment, marked as answer]
Hi there. As of the last commit, the extension should now be compatible with linux/windows/wsl. Please give it a try!
